src/config.js

```diff
@@ -174,8 +177,12 @@ module.exports.setConfig = function setConfig(configData) {
         config.disableHttpResponseBodyCapture = configData.disableHttpResponseBodyCapture;
     }
 
-    if (configData.ignoredKeys) {
-        config.ignoredKeys = configData.ignoredKeys;
+    if (configData.ignoredKeys && Array.isArray(configData.ignoredKeys)) {
+        const filteredIgnoredKeys = configData.ignoredKeys.filter(key => typeof key !== 'function');
+        if (filteredIgnoredKeys.length !== configData.ignoredKeys.length) {
+            console.warn('Epsagon deprecation warning: function ignoredKeys have been deprecated and will be ignored'); // eslint-disable-line no-console
```

Comment:
I saw there's a `utils.printWarning` function, maybe use that? 
and in any case, add the problematic ignored keys to make this usable.